demo/main.py

```diff
@@ -1,24 +1,27 @@
-import asyncio
 import time
 import base64
 from cryptography import fernet
 from aiohttp import web
-from aiohttp_session import setup, get_session, session_middleware
+from aiohttp_session import setup, get_session
 from aiohttp_session.cookie_storage import EncryptedCookieStorage
 
-@asyncio.coroutine
-def handler(request):
-    session = yield from get_session(request)
+
+async def handler(request):
+    session = await get_session(request)
     last_visit = session['last_visit'] if 'last_visit' in session else None
     session['last_visit'] = time.time()
     text = 'Last visited: {}'.format(last_visit)
-    return web.Response(body=text.encode('utf-8'))
+    return web.Response(body=text)
```

Comment:
No. Either `body=text.encode()` or `text=text`

Reply:
Just saw this and submitted a change. Sorry about that!

Reply:
Firefox kept downloading the file, and I couldn't figure out why... I had to set `content_type='text/html'` or use `text=` as it sets a nice content_type for us.